### ISSUE TITLE
Fix FN S6265: BucketDeployment import library name

### ISSUE THREAD

**Issue by pedro-oliveira-sonarsource** [SonarSource/SonarJS#3663]
### **What**

In  #3198 , support for _aws-cdk-lib.aws_s3.Bucket_ was added, where we raise an issue if a bucket is configured to be accessible publicly.

On a validation sprint, when analyzing the [code sample](https://github.com/SonarSource/security-expected-issues/pull/721/files) produced during the research for rule specification, FNs were detected.

With some further digging, it appears the method identifier for _BucketDeployment_ _isS3BucketDeploymentConstructor_ is using an incorrect module name, [see](https://github.com/SonarSource/SonarJS/pull/3303/files#diff-6c4906a965f2f643c10de724bbcd4bd6404cdaac1b782fff0aaeb1080c514f5cR83):

`BucketDeployment` is imported from `aws-s3-deployment` not `aws-s3`.

**Manipulated True-Positive - Not Executable**
```
const s3 = require('aws-cdk-lib/aws-s3'); // <------------

new s3.BucketDeployment(this, 'upload', {
    sources: [s3deploy.Source.asset(path.join(__dirname, '../lib'))],
    destinationBucket: bucketNoncompliant4,
    accessControl: s3.BucketAccessControl.PUBLIC_READ_WRITE // TP - Sensitive: Make sure granting [PUBLIC_READ|PUBLIC_READ_WRITE|AUTHENTICATED_READ] access is safe here.
});
```
**False-Negative**
```
const s3deploy  = require('aws-cdk-lib/aws-s3-deployment'); // <------------

new s3deploy.BucketDeployment(this, 'upload', {
    sources: [s3deploy.Source.asset(path.join(__dirname, '../lib'))],
    destinationBucket: bucketNoncompliant4,
    accessControl: s3.BucketAccessControl.PUBLIC_READ_WRITE // FN - Sensitive: Make sure granting [PUBLIC_READ|PUBLIC_READ_WRITE|AUTHENTICATED_READ] access is safe here.
});
```
